src/test/kotlin/persistence/data/LanguageStore.kt

```diff
@@ -0,0 +1,24 @@
+package persistence.data
+
+import data.model.Language
+
+class LanguageStore {
+    companion object {
+        val languages: List<Language> = listOf(
```

Comment:
Right now we are just doing `returnedObservable.blockingFirst()` in the tests. We could compare with observables, but I'm not sure if that would work since we really want to compare the contents of the observable.